### DISCUSSION TITLE
Tagging interfaces to LAG

### DISCUSSION THREAD

**Discussion by xavbroadcast** [netbox-community/netbox#17314] [Help Wanted!] [answered]
Netbox 4.0.9 / 4.10, I can no longer create a LAG type interface and then link physical interfaces to it.

Previously, I'd create an interfae of type LAG. Then I'd go to a normal physical interface, edit it, and the LAG option was in the list of boxes with the ability to assign it to a LAG.

This seems to have gone. I've also just looked at the public demo version of netbox and have the same.

On my main system which I've had running and upgraded from earlier versions of NetBox, all is OK.

I've also noticed that the LAG is no longer an available column in the interface list. It's as if it's just not in the database ?
![image](https://github.com/user-attachments/assets/19b8885f-2601-4645-8ac9-8e95b778143e)

Bridge interface is there, just LAG isn't.

https://demo.netbox.dev/dcim/device-types/21/interfaces/

What am I missing? Or is it a bug?

**Comment by xavbroadcast** [comment]
Forgot to add I've also checked the permissions in case it was missing, but even with everything allowed, it's still not visible. I wonder if it's a default permissions that's somehow broken in recent releases where the database is created from scratch.

**Comment by candlerb** [comment, marked as answer]
> https://demo.netbox.dev/dcim/device-types/21/interfaces/
> 
> What am I missing? Or is it a bug?

I think what you are missing is that you are looking at an **interface template** on a **device type**, not an **interface** on a **device**.

I don't know if it's ever been possible to configure interface to LAG associations on a **device type**. In principle it could be, but it would be an odd thing to do, as manufacturers don't supply switches with interfaces already linked to LAGs.

**Reply by candlerb** [reply]
FYI I created a device called "dell1" from your device type, and I can associate physical interfaces with a LAG:

![image](https://github.com/user-attachments/assets/4bef2ef7-0588-48c1-acbb-078d8255219e)

**Reply by xavbroadcast** [reply]
Doh!
I was convinced I had done it in the past, but you're right, it must have been on actual devices and not templates.

> I don't know if it's ever been possible to configure interface to LAG associations on a device type. In principle it could be, but it would be an odd thing to do, as manufacturers don't supply switches with interfaces already linked to LAGs.

I guess you can't as that what I was trying to do. I do think however it would be useful, often we have several servers for example that have templated configurations (images rolled out) with complex network configurations, some including LAGs, and all of which are supported by Netbox. But currently, we have to manually create this individually on each device we add. If it could be included in a device type template, that would save a lot of time.
